grpc/python/BUILD

```diff
@@ -36,14 +36,14 @@ python_grpc_compile(
 )
 
 python_repackage(
-    name = "protocol_pkg",
-    src = ":protocol_src",
+    name = "protocol-pkg",
+    src = ":protocol-src",
     package = "grakn_protocol"
 )
 
 py_library(
     name = "protocol",
-    srcs = ["protocol_pkg"],
+    srcs = ["protocol-pkg"],
```

Comment:
Renamed for consistency on the way past

Reply:
Nice effort!

However, will a Bazel target with `-` rather than `_` work in Python @vmax ?